### PR TITLE
new script utils/check_import_finished.php

### PR DESCRIPTION
**Pull request by mtmail**
For https://github.com/openstreetmap/Nominatim/issues/1574

Both number of checks and output in case of failures can still be improved.

### CODE REVIEW

File: utils/check_import_finished.php [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    // sql/indices.src.sql
+    'idx_word_word_id',
+    'idx_place_addressline_address_place_id',
+    'idx_placex_rank_search',
+    'idx_placex_rank_address',
+    'idx_placex_pendingsector',
+    'idx_placex_parent_place_id',
+    'idx_placex_geometry_reverse_lookupPoint',
+    'idx_placex_geometry_reverse_lookupPolygon',
+    'idx_placex_geometry_reverse_placeNode',
```
Comment (lonvia) [inline utils/check_import_finished.php]: These three indexes have lower case for the last word. ..._lookuppoint, ..._lookuppolygon etc.

File: utils/check_import_finished.php [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        echo <<< END
+        Hints:
+        * Rerun the setup.php --create-search-indices step
+END;
+        exit(1);
+    }
+}
+
+echo 'Checking indexing status ... ';
+if ($oDB->tableExists('import_status')) {
```
Comment (lonvia) [inline utils/check_import_finished.php]: Better thing to check here: SELECT count(*) FROM placex where indexed_status > 0
That should be 0 when indexing is finished.
Also: I'd move that before the index checks as this step is before indexing during import.

Comment (mtmail) [reply utils/check_import_finished.php]: changed and moved

File: utils/check_import_finished.php [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    $print_fail();
+    echo <<< END
+    * The import didn't finish.
+    Hints:
+    * Check the output of the utils/setup.php you ran.
+    Usually the osm2pgsql step failed. Check for errors related to
+    * the file you imported not containing any places
+    * harddrive full
+    * out of memory (RAM)
+    * osm2pgsql killed by other scripts, the for consuming to much memory
```
[the remark this answers is not in the document]
Comment (mtmail) [reply utils/check_import_finished.php]: removed

**Comment by lonvia** [comment]
Merged as 7db0da40ad02694b82fe9a387f277a58ca7c5fce where I've removed the accidental addition of nearby.php to the CMake script.

**Pull request closed without merging**